kratos/solving_strategies/schemes/residual_implicit_time_scheme.h

```diff
@@ -0,0 +1,544 @@
+//    |  /           |
```

Comment:
shoulnd't this file be named `residual_based_implicit_time_scheme`?

Reply:
Yep